Marlin/Marlin_main.cpp

```diff
@@ -151,6 +151,7 @@
 // M207 - Set retract length S[positive mm] F[feedrate mm/min] Z[additional zlift/hop], stays in mm regardless of M200 setting
 // M208 - Set recover=unretract length S[positive mm surplus to the M207 S*] F[feedrate mm/sec]
 // M209 - S<1=true/0=false> enable automatic retract detect if the slicer did not support G10/11: every normal extrude-only move will be classified as retract depending on the direction.
+// M211 - Disable software endstops. This allows using negative coordinates for milling.
```

Comment:
You can already use negative coordinates. You should make it clear this means something else, like being allowed to go below `Z_MIN_POS`, even if it is set to -20.0mm.